src/azure-cli/azure/cli/command_modules/resource/tests/latest/bicep_simple_template.bicep

```diff
@@ -0,0 +1,12 @@
+param location string = 'centralus'
+
+@minLength(3)
+@maxLength(24)
+param storageAccountName string = 'uniquestorage001' // must be globally unique
+
+resource stg 'Providers.Test/statefulResources@2014-04-01' = {
+    name: storageAccountName
+    location: location
+}
```

Comment:
Please help move all the `bicep` files used for testing to the `src/azure-cli/azure/cli/command_modules/resource/tests/latest/data` folder, so that our packaging tests of release CI pipeline can load these files correctly.